beacon-chain/forkchoice/protoarray/proposer_boost_test.go

```diff
@@ -88,8 +88,8 @@ func TestForkChoice_BoostProposerRoot_PreventsExAnteAttack(t *testing.T) {
 		//         2
 		//         |
 		//         3 <- HEAD
-		slot = types.Slot(2)
-		newRoot = indexToHash(2)
+		slot = types.Slot(3)
+		newRoot = indexToHash(3)
```

Comment:
Why this change?

Reply:
This test case in particular was not matching the comments, and not caught in coverage because it doesn't make a difference to proposer boost if we did slot 2 or 3, but my comments say it should be slot 3. The behavior tested is still the same, but made it 3 to align more with my comment